docs/setup/keadm.md

```diff
@@ -95,6 +95,48 @@ Host has mosquit+ already installed and running. Hence skipping the installation
 KubeEdge edgecore is running, For logs visit:  /var/log/kubeedge/edgecore.log
 ```
 
+### Enable `kubectl logs` Feature
```

Comment:
logs and metric feature ?

@fisherxu PTAL

Reply:
@kadisi ,`metric` may need wait for a new release of `metric-server`. Otherwise user need to make a choice between `kubelet` or `edgecore`.